CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.13.0 - 2016-09-07
+
+* Improved environment variable HERMIONE_SKIP_BROWSERS. Now skipped tests are shown in the report.
```

Comment:
Show skipped tests in the report if they was skipped by the HERMIONE_SKIP_BROWSERS environment variable